imxrt-hal/src/iomuxc.rs

```diff
@@ -81,59 +85,64 @@ pub struct IOMUXC {
     pub gpio_emc_08: gpio::GPIO_EMC_08<Alt5>,
     pub gpio_emc_31: gpio::GPIO_EMC_31<Alt5>,
     pub gpio_emc_32: gpio::GPIO_EMC_32<Alt5>,
-    //
+
     // GPRs
-    //
     pub gpr: GPR,
 }
 
 impl IOMUXC {
-    pub(crate) fn new(iomuxc: crate::ral::iomuxc::IOMUXC) -> Self {
+
+    #[cfg(feature = "imxrt106x")]
```

Comment:
Should this `#[cfg(feature = "imxrt106x")]` go on top of the `impl IOMUXC`? It looks like we have another one on top of the `struct IOMUXC` declaration. Would the possibly-empty `impl IOMUXC` compile without a `struct IOMUXC`?

Reply:
Yeah I wasn't sure how I wanted to deal with this exactly, I want to maybe even import a different iomux per family, like in lib.rs 
```
#[cfg(feature = "imxrt106x")]
use iomuxc/imxrt106x as iomuxc;

So then the peripherals struct doesn't have to deal with it, similar to how ral exposes things based on feature flags. It's something I thought was worth discussing. Let me know what you think on that.